.goreleaser.yml

```diff
@@ -267,3 +267,29 @@ archives:
 checksum:
   name_template: "checksums.txt"
   algorithm: sha256
+
+# signs the checksum file
+# https://goreleaser.com/customization/sign
+signs:
+- cmd: cosign
+  stdin: '{{ .Env.COSIGN_PWD }}'
+  args:
+    - sign-blob
+    - '--key=cosign.key'
+    - '--output-certificate=${certificate}'
+    - '--output-signature=${signature}'
+    - '${artifact}'
+  artifacts: all
+  output: true
+
+# signs our docker image
+# https://goreleaser.com/customization/docker_sign
+docker_signs:
+  - cmd: cosign
+    artifacts: all
+    output: true
+    stdin: '{{ .Env.COSIGN_PWD }}'
+    args:
+    - 'sign'
+    - '--key=cosign.key'
+    - '${artifact}'
```

Comment:
Please format file correctly so that looks consistent across. 

I see different indentation levels for signs and docker_signs list.

Also add filed ending line.